test/backend/UPathTestSuite.scala

```diff
@@ -0,0 +1,130 @@
+package backend
+
+import com.scalableminds.webknossos.datastore.helpers.UPath
+import org.scalatestplus.play.PlaySpec
+
+class UPathTestSuite extends PlaySpec {
+
+  "UPath" should {
+    "Be constructable from well-formed string" in {
+      assert(UPath.fromString("relative/elsewhere").exists(_.toString == "./relative/elsewhere"))
+      assert(UPath.fromString("./relative/elsewehere").exists(_.toString == "./relative/elsewehere"))
+      assert(UPath.fromString("/absolute/somewhere").exists(_.toString == "/absolute/somewhere"))
+      assert(UPath.fromString("/absolute/some$where").exists(_.toString == "/absolute/some$where"))
+      assert(UPath.fromString("/absolute/with²Unicode").exists(_.toString == "/absolute/with²Unicode"))
+      assert(UPath.fromString("/absolute/ with space").exists(_.toString == "/absolute/ with space"))
+      assert(UPath.fromString("file:///absolute/with²Unicode").exists(_.toString == "/absolute/with²Unicode"))
```

Comment:
So toString does not print the scheme? This might get confusing in the debug / server logs when the schem is missing :thinking:?

Edit: Oh just saw below: So this only goes for local files / paths? In that case, this should be fine I think

Reply:
Exactly, local paths will always be printed starting with `./` or `/` (but parsing is more permissive). Remote paths will always be printed with scheme.